src/instructlab/model/chat.py

```diff
@@ -219,6 +231,10 @@ def chat(
             ctx.obj.config.serve, model_path=model
         )
 
+        if serving_log_file:
+            logger.debug(f"Setting serving log file to {serving_log_file}")
+            backend_instance.set_log_file(serving_log_file)
```

Comment:
Wondering whether this would be better passed to run_detached or to select_backend?

Reply:
Makes more sense indeed. Thanks!